### PR TITLE
FAQ: Where can I buy Monero for fiat?

### LINKED ISSUE #941: Add FAQ: "Where can i buy monero for fiat?"
This question gets asked like 3 times a week on /r/monero. Worth making a dedicated answer or maybe we should just improve [How can i get Monero?](https://web.getmonero.org/get-started/faq/#anchor-buy) to include more info for the conversion from fiat. A mention to how to convert BTC -> XMR is probably good to have, since that's another very common question.

### PR DESCRIPTION
**Pull request by dginovker**
Closes #941 

I feel like the table could look prettier, but I'm not sure what CSS properties to add to it. If someone else wants to edit those (or suggest which ones to try), I'm all ears.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ If you need support about something related to the website, plese join `#monero-
 Once you have the above list of things, it's typically a good idea to build the website from your local computer to make sure it works before you make any changes. To do this, complete the following steps:
 
 1. Navigate to your local `monero-site` repository.
-2. Serve the website: `bundle exec jekyll serve --baseurl ''`. If you want, you can speedup thi process by loading only the last blog post instead of all of them. Simply add `--limit_posts 1` to the command above. The resulting command will be `bundle exec jekyll serve --limit_posts 1 --baseurl ''`.
+2. Serve the website: `bundle exec jekyll serve --baseurl ''`. If you want, you can speedup the process by loading only the last blog post instead of all of them. Simply add `--limit_posts 1` to the command above. The resulting command will be `bundle exec jekyll serve --limit_posts 1 --baseurl ''`.
```
Comment (erciccione) [inline README.md:39]: The changes to the readme are not related to the commits. Please remove them or edit the commit description

File: _i18n/en.yml [hunk 2]
```diff
@@ -378,6 +378,52 @@ faq:
   a13: Monero is not magic. If you use Monero but give your name and address to another party, the other party will not magically forget your name and address. If you give out your secret keys, others will know what you've done. If you get compromised, others will be able to keylog you. If you use a weak password, others will be able to brute force your keys file. If you backup your seed in the cloud, you'll be poorer soon.
   q14: Is Monero 100% anonymous?
   a14: There is no such thing as 100% anonymous. If nothing else, your anonymity set is the set of people using Monero. Some people don't use Monero. Monero may also have bugs. Even if not, ways may exist to infer some information through Monero's privacy layers, either now or later. Attacks only get better. If you wear a seatbelt, you can still die in a car crash. Use common sense, prudence and defense in depth.
+  qwherebuy: Where can I buy Monero for Fiat?
+  awherebuy: |
```
Comment (erciccione) [inline _i18n/en.yml:382]: The `|` doesn't get parsed corectly by weblate, better use `>`

File: _i18n/en.yml [hunk 3]
```diff
@@ -378,6 +378,52 @@ faq:
...
+      Keep in mind the following table is incomplete, and there may be more exchanges out there than what are listed below.
+
+      <table>
```
Comment (erciccione) [inline _i18n/en.yml:385]: This section doesn't need to be in the variable. Keep in mind that all these strings would be parsed by weblate, which will pull all the HTML with all those strings and show it to the translators, when the strings that would actually need to be translated are only 3 (1 before the table, 2 after).

**Comment by erciccione** [comment]
ping @dginovker

**Comment by dginovker** [comment]
I won't be changing this commit until at least the weekend - I'll close for now

**Pull request closed without merging**